packages/carbon-graphs/src/js/controls/Graph/helpers/helpers.js

```diff
@@ -264,6 +264,66 @@ const getAxesLabelCharacterLimits = (config) => ({
   xAxisLimit: getXAxisWidth(config) / 7.33,
   yAndY2AxisLimit: config.height / 8,
 });
+/**
+ * Adds onClick event for each label in Axes.
+ * Criteria:
+ *  * Text needs to have a valid function for onClick
+ *  * Label should exceed the calculated character limit with respect to axis length
+ *
+ * @private
+ * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
+ * @param {string} className - class name of axis
+ * @param {object} axisObj - Axis object provided by the consumer
+ * @param {number} charLimit - character limit of label in axis
+ * @param {string} axisType - type of axis
+ * @returns {object} d3 svg path
+ */
+const addLabelEventHandler = (
+  canvasSVG,
+  className,
+  axisObj,
+  charLimit,
+  axisType,
+) => canvasSVG.selectAll(`.${className} text`).each(function () {
+  // eslint-disable-next-line no-unused-expressions
```

Comment:
If this is necessary, then I would move this to the top of the file instead of repeating it.

Reply:
It is just for a line and we are not using this kind of disabling lint anywhere in this file, that is why I just left it here.